### PR TITLE
Add configuration for Grafana dashboard and Postgres replicator

### PR DESCRIPTION
**Pull request by brandonpaiz**
The code here will configure the Postgres replicator from test-infra (PR https://github.com/grpc/test-infra/pull/244) and a Grafana instance. Secrets are pulled from the GCP secret manager and passed to the container as environment variables.

### CODE REVIEW

File: tools/run_tests/performance/dashboard_config/cloudbuild.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+steps:
```
Comment (jtattermusch) [inline tools/run_tests/performance/dashboard_config/cloudbuild.yaml:1]: what is this cloudbuild.yaml file for?

File: tools/run_tests/performance/dashboard_config/grafana_config/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 gRPC authors.
```
Comment (jtattermusch) [inline tools/run_tests/performance/dashboard_config/grafana_config/Dockerfile:1]: nit: why do you have the `_config` suffix after almost all directory names? I think it doesn't add much value.

File: tools/run_tests/performance/dashboard_config/cloudbuild.yaml (hunk 1 above)
Comment (jtattermusch) [reply tools/run_tests/performance/dashboard_config/cloudbuild.yaml:1]: I think the tools/run_tests/performance is not a good place to put this. 
It is where our legacy performance benchmarks live and we don't want to add more stuff there. I'm actually not convinced that the configuration belongs to the grpc/grpc repository at all

**Review by jtattermusch — COMMENTED** [review]
I think grpc/test-infra might actually be a better place to put this. I don't think we gain much by adding the deployment config into grpc/grpc (which is already a very large repo that has hundreds of things). What are the main reasons why you wanted to put the config in grpc/grpc and no it grpc/test-infra (which is where the corresponding code lives).

File: tools/run_tests/performance/dashboard_config/cloudbuild.yaml (hunk 1 above)
Comment (brandonpaiz) [reply tools/run_tests/performance/dashboard_config/cloudbuild.yaml:1]: This is the cloud build configuration for automated deployment

**Comment by brandonpaiz** [comment]
Config files are to be added in this PR instead: https://github.com/grpc/test-infra/pull/244

**Pull request closed without merging**